source/app/viewoptions.h

```diff
@@ -19,11 +19,18 @@
 #define APP_VIEWOPTIONS_H
 
 #include <optional>
+#include <array>
+#include <algorithm>
 
 /// Stores any view options that are not saved with the score (e.g. the current
 /// zoom level or the active score filter).
 class ViewOptions
 {
+public:
+    static constexpr double MIN_ZOOM = 25;
+    static constexpr double MAX_ZOOM = 300;
+    static constexpr std::array<unsigned short, 14> ZOOM_LEVELS = { 25, 50, 70, 80, 90, 100, 110, 125, 150, 175, 200, 225, 250, 300 };
```

Comment:
Wanted these to be scoped. Wasn't sure whether to use namespace or the class though